time_tracker_api/projects/projects_model.py

```diff
@@ -44,3 +46,15 @@ def __init__(self):
             SQLCRUDDao.__init__(self, ProjectSQLModel)
 
     return ProjectSQLDao()
+
+
+container_definition = {
+    'id': 'project',
+    'partition_key': PartitionKey(path='/tenant_id'),
+    'unique_key_policy': {
```

Comment:
I think we need project_type and maybe customer_id

Reply:
No, you can actually have multiple projects with the same type in a same tenant. That is not a problem.